src/app/pages/list/action-list/action-list.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-toolbar color="primary">
+    <div>Action List</div>
+    <div fxFlex style="margin: auto;"></div>
+    <button mat-icon-button (click)="onAddItem()"><mat-icon>add</mat-icon></button>
+</mat-toolbar>
+<mat-nav-list *ngIf="data.length > 0">
+    <pxb-info-list-item
+        [hidePadding]="true"
```

Comment:
the padding on these doesn't look hidden, but that might be an issue with the ILI component...are we using the latest version?

Reply:
I think updating to @pxblue/angular-components ^2.0.0 will fix the padding issue.